pytket/pytket/circuit/display/__init__.py

```diff
@@ -71,9 +71,68 @@ def _render(self, filename: str) -> markupsafe.Markup:
 class CircuitRenderer:
     """Class to manage circuit rendering within a given jinja2 environment."""
 
+    _ALLOWED_RENDER_OPTIONS = {
+        "zx_style": "zxStyle",
+        "condense_c_bits": "condenseCBits",
+        "recursive": "recursive",
+        "condensed": "condensed",
+        "dark_theme": "darkTheme",
+        "transparent_bg": "transparentBg",
+        "crop_params": "cropParams",
+    }
+    zx_style: Optional[bool] = None
+    condense_c_bits: Optional[bool] = None
+    recursive: Optional[bool] = None
+    condensed: Optional[bool] = None
+    dark_theme: Optional[bool] = None
+    transparent_bg: Optional[bool] = None
+    crop_params: Optional[bool] = None
+
+    _ALLOWED_CONFIG_OPTIONS = ["min_height", "min_width"]
+    min_height: str = "400px"
+    min_width: str = "500px"
+
     def __init__(self, env: Environment):
         self.env = env
 
+    def set_render_options(self, **kwargs: Union[bool, str]) -> None:
+        """
+        Set rendering defaults.
+
+        :param min_height: str, initial height of circuit display.
```

Comment:
Looks good to me, thanks :) 

I just wanted to check how the defaults are configured.

 If I use `set_render_options` in a notebook environment does this override the defaults for all other diagrams that I would render afterwards? Would restarting the jupyter kernel revert back to the default settings?

Reply:
Yeah. It overrides the defaults for all diagrams you render afterwards using that instance (ie calling `circuit_renderer.render_circuit_jupyter`),but doesn't affect eg the default `render_circuit_jupyter` you can import directly.